index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    
+    
+</head>
+<body>
+  <header>
+   <div class="topnav">
+      <a class = "logo">Bobson</a>
```

Comment:
- [ ] Hey @KALUNGI-J-BOBSON great job implementing the Figma template design. However, the anchor tag for your logo is missing the `href` attribute. Kindly fix that to make your logo clickable.